server/integration_tests/test_data/international/query_3/chart_config.json

```diff
@@ -4,38 +4,6 @@
     "categories": [
       {
         "blocks": [
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "Amount_EconomicActivity_GrossDomesticProduction_Nominal"
-                    ],
-                    "title": "GDP (Nominal Value) in Eurostat Nuts 3 Places of United Kingdom (${date})",
-                    "type": "MAP"
-                  }
-                ]
-              },
-              {
```

Comment:
Wonder what happened here?

I wonder if this is from @ajaits AA2 change?  This doesn't seem good.